frontend/public/components/build-config.tsx

```diff
@@ -86,30 +87,67 @@ export const BuildConfigsDetailsPage: React.SFC<BuildConfigsDetailsPageProps> =
     pages={pages} />;
 BuildConfigsDetailsPage.displayName = 'BuildConfigsDetailsPage';
 
-const BuildConfigsHeader = props => <ListHeader>
-  <ColHead {...props} className="col-sm-3 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-3 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="metadata.labels">Labels</ColHead>
-  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
-</ListHeader>;
+const tableColumnClasses = [
+  classNames('pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  classNames('pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  Kebab.columnClass,
+];
 
-const BuildConfigsRow: React.SFC<BuildConfigsRowProps> = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-sm-3 col-xs-6">
-    <ResourceLink kind={BuildConfigsReference} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
-  </div>
-  <div className="col-sm-3 col-xs-6 co-break-word">
-    <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
-  </div>
-  <div className="col-sm-3 hidden-xs">
-    <LabelList kind={BuildConfigsReference} labels={obj.metadata.labels} />
-  </div>
-  <div className="col-sm-3 hidden-xs">
-    { fromNow(obj.metadata.creationTimestamp) }
-  </div>
-  <div className="dropdown-kebab-pf">
-    <ResourceKebab actions={menuActions} kind={BuildConfigsReference} resource={obj} />
-  </div>
-</div>;
+export const BuildConfigsTableHeader = () => {
+  return [
+    {
+      title: 'Name', sortField: 'metadata.name', transforms: [sortable],
+      props: { className: tableColumnClasses[0]},
+    },
+    {
+      title: 'Namespace', sortField: 'metadata.namespace', transforms: [sortable],
+      props: { className: tableColumnClasses[1]},
+    },
+    {
+      title: 'Labels', sortField: 'metadata.labels', transforms: [sortable],
+      props: { className: tableColumnClasses[2]},
+    },
+    {
+      title: 'Created', sortField: 'metadata.creationTimestamp', transforms: [sortable],
+      props: { className: tableColumnClasses[3]},
+    },
+    { title: '',
+      props: { className: tableColumnClasses[4]},
+    },
+  ];
+};
+BuildConfigsTableHeader.displayName = 'BuildConfigsTableHeader';
+
+export const BuildConfigsTableRow: React.FC<BuildConfigsTableRowProps> = ({obj, index, key, style}) => {
```

Comment:
Unnecessary export. None of the header or row components should need to be exported afaik

Reply:
again, possibly useful for tests. I can certainly remove. What's your stance?

Reply:
fixed in https://github.com/openshift/console/pull/1465/commits/6e61b79af85a9fe944a93665a59d3418e9b958d5